### DISCUSSION TITLE
FOOTSTEP FPS GAMES

### DISCUSSION THREAD

**Discussion by deddroid** [wwmm/easyeffects#3474] [General]
hi! anyone have tryed to improve the footsteps audio in fps games? I usually play in apex legends, where footsteps are so silent.

I search online and found "loudnessCrystalEqualizer" that works better than stock audio, but my question is: Is there a way to improve audio for gaming? ty

**Comment by wwmm** [comment]
I never tried. The processing I apply to games is the usual generic dynamic range compression. Maybe just applying some compression combined with the autogain will be enough to make footsteps more noticeable. More than that will probably require to figure out the average frequency of the footsteps and trying to either boost it or adding a multiband compressor where compression is set to the other frequencies bands while trying to keep the band where the footsteps are located alone. In any case prepare yourself for a lot of trial and error attempts :smile:

**Reply by deddroid** [reply]
thanks for your answer and for your advice, it will be very useful to me!  😁

 I'll test a lot of different options, and I'll report here if found something usefull for this project

**Reply by pinbuck** [reply]
> thanks for your answer and for your advice, it will be very useful to me! 😁
> 
> I'll test a lot of different options, and I'll report here if found something usefull for this project

techyes city has a guide on this, would recommend that
